### PR TITLE
Backend

### PR DESCRIPTION
**Pull request by lwangwan**

### CODE REVIEW

File: flaskProject/resources/motos.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+from models.moto_model import MotoModel
+
+
+class Moto(Resource):
+    def get(self, id):
+        try:
+            moto = MotoModel.find_by_id(id)
+            return {'moto':moto.json()}, 200
+        except:
+            return {"message": "Error Get Moto"}, 500
```
Comment (rchacoma8) [inline flaskProject/resources/motos.py:11]: El mensaje debería ser más aclaratorio para el usuario. Como: return {'message': "Moto with id [{}] Not Found".format(id)}, 404

File: flaskProject/resources/motos.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        data = parser.parse_args()
+
+        try:
+            moto = MotoModel(data['state'], data['matricula'], data['date_estreno'], data['model_generic'], data['last_coordinate_latitude'],
+                               data['last_coordinate_longitude'], data['km_restantes'], data['km_totales'],
+                              data['date_last_check'])
+            MotoModel.save_to_db(moto)
+            return {"message": "Moto added successfully"}, 200
+        except:
+            return {"message": "Error Post Moto"}, 500
```
Comment (rchacoma8) [inline flaskProject/resources/motos.py:33]: El mensaje debería ser más aclaratorio para el usuario. Como: "Error adding the moto in the database"

File: flaskProject/resources/motos.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+    def delete(self, id):
+        try:
+            moto = MotoModel.find_by_id(id)
+            if moto:
+                MotoModel.delete_from_db(moto)
+                return {"message": "Moto deleted"}, 200
+            return {"message": "Moto not found "}, 404
```
Comment (rchacoma8) [inline flaskProject/resources/motos.py:41]: return {'message': "Moto with id [{}] Not Found".format(id)}, 404

**Review by edumargra — APPROVED** [review]
There doesn't seem to be any breaking-type problems; although the Article "thing" doesn't seem to be necessary, and there seems to be some "quirky" code... A little clean-up could be done, but it could be done later.

**Review by rchacoma8 — COMMENTED** [review]
Technical Debts de mensajes y listas

File: flaskProject/resources/client.py [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            c.email = data['email']
+            c.iban = data['iban']
+            c.save_to_db()
+
+            return c.json(), 200
+        else:
+            return {'message': 'There is no client with ID [{}] .'.format(client_id)}, 404
+
+class ClientsList(Resource):
+    def get(self):
```
Comment (rchacoma8) [inline flaskProject/resources/client.py:69]: This method could be done with a comprehension list

**Review by rchacoma8 — APPROVED** [review]

**Pull request merged**